Documentation.md

```diff
@@ -7,35 +7,54 @@ This documentation is intended to guide you in developing, compiling, debugging
 [You can use this script (Linux, run in Qemu)](https://github.com/ringwormGO-organization/ringOS/blob/main/ringOS-beta.sh)
 
 ### Software you need
-1. [Qemu recommended](#install-qemu) or [Virtual Box, VmWare](#running-in-virtual-box-or-vmware)
+1. [Qemu recommended](#install-qemu) or [Virtual Box, VMWare Workstation](#running-in-virtual-box-or-vmware)
 2. WSL/WSL2 (If developing on Windows) or Linux (recommended)
 
 ### Type of release for running in Qemu
 Download a Zip file of Qemu release (example: https://github.com/Andrej123456789/ringOS/releases/tag/Beta11_Qemu).
 
 Unzip the downloaded Zip folder and navigate to the contents in your Terminal (Linux) or using WSL/WSL2 on Windows ('cd')
 
+Support for Red Hat(-based) Linux distribution just come, if you want to improve consider become a contributor.
+
 --------------------------------------------------------------------------------------------------------------------------------
 ### Compile beta version from shell script
 #### How to run shell script for Debian based Linux distros:
 `./ringOS-beta.sh`
 
-If that don't work, type: `chmod +x ringOS-beta.sh` and try again.
```

Comment:
Why this is removed?

Reply:
Add `chmod +x ./ringOS-beta.sh`

Reply:
People will know to change the exec bits before running an executable shell script.

Reply:
Are you sure?

Reply:
Yes, it is the most basic Linux computer stuff that anyone use it should know